docs/user-guide/configure-map.md

```diff
@@ -0,0 +1,52 @@
+# Configure the map
+**********************
+
+With the Edit map configuration button <img src="../img/button/configure-map-button.jpg" class="ms-docbutton"/> the **Map Inline Editor** opens to give the opportunity to do quick customizations (like basic map settings, order of layers, layer opacity and something more) to the map (more advanced customizations then, are allowed only through the *Advanced Map Editor*).
```

Comment:
With the Edit map configuration button <img src="../img/button/configure-map-button.jpg" class="ms-docbutton"/> the **Map Inline Editor** opens to give the opportunity to do quick customizations (like basic map settings, layer opacity and something more) to the map (more advanced customizations then, are allowed only through the *Advanced Map Editor*).